### PR TITLE
Add measles blog post

### PR DESCRIPTION
**Pull request by kimandrews**
## Description of proposed changes

This PR adds a blog post about new measles resources.

## Checklist

- [ ] Checks pass
- [ ] Check if changes affect the [resource index JSON revision](https://docs.nextstrain.org/projects/nextstrain-dot-org/en/latest/resource-collection.html#resource-index-revisions)

### CODE REVIEW

**Comment by genehack** [comment]
[Preview of the post](https://nextstrain-s-add-measle-sd1slr.herokuapp.com/blog/2024-06-12-new-resources-for-measles)

**Comment by genehack** [comment]
(probably non-blocking) 

I note that the [mpox dataset](https://nextstrain.org/mpox/clade-IIb) has "Enabled by data from [GenBank](https://www.ncbi.nlm.nih.gov/genbank/)." but [measles](https://nextstrain.org/measles/genome) does not — do we consistently give attribution like this? (Seems nice.)

File: static-site/content/blog/2024-06-12-new-resources-for-measles.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+1. Full genome sequences
+1. 450bp region of the N gene ("N450")
+
+In both phylogenies, sequences are classified to genotype using the [nomenclature system established by the World Health Organization](https://www.who.int/publications/i/item/WER8709).
+
+As part of this work, we developed an [open-source tool for classifying measles genotypes](https://clades.nextstrain.org/?dataset-name=nextstrain/measles/N450/WHO-2012). This tool is a [Nextclade dataset](https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html), which allows users to drag-and-drop sequences onto a web browser to obtain genotype classifications. It also outputs sequence quality control metrics and displays user sequences on a phylogenetic tree.
+
+We also provide daily-updated links where all NCBI GenBank measles [sequences](https://data.nextstrain.org/files/workflows/measles/sequences.fasta.zst) and [metadata](https://data.nextstrain.org/files/workflows/measles/metadata.tsv.zst) can be downloaded. These currently include over 23,000 sequences, of which ~1,000 are full genome sequences, and ~19,000 include the N450 region.
```
Comment (genehack) [inline static-site/content/blog/2024-06-12-new-resources-for-measles.md:16]: "daily-updated" is (IMO) kinda clunky, how about: 

```suggestion
We also provide links, updated daily, where all NCBI GenBank measles [sequences](https://data.nextstrain.org/files/workflows/measles/sequences.fasta.zst) and [metadata](https://data.nextstrain.org/files/workflows/measles/metadata.tsv.zst) can be downloaded. These currently include over 23,000 sequences, of which ~1,000 are full genome sequences, and ~19,000 include the N450 region.
```

File: static-site/content/blog/2024-06-12-new-resources-for-measles.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+With measles cases currently increasing globally, we expect these tools to be useful resources for tracking the spread of this viral pathogen. This work is made possible by the open sharing of genetic data by research groups from all over the world. We gratefully acknowledge their contributions.
```
Comment (genehack) [inline static-site/content/blog/2024-06-12-new-resources-for-measles.md:18]: > With measles cases currently increasing globally

I wonder if it's worth finding and linking an authoritative source here.

**Review by genehack — APPROVED** [review]
couple minor things; none of them need to be blocking, take or leave. :grin:

File: static-site/content/blog/2024-06-12-new-resources-for-measles.md (hunk 1 above)
Comment (kimandrews) [reply static-site/content/blog/2024-06-12-new-resources-for-measles.md]: Done in [5bddcd7](https://github.com/nextstrain/nextstrain.org/pull/911/commits/5bddcd7659cfa2d55712dc2465d39c04bd18291f)

File: static-site/content/blog/2024-06-12-new-resources-for-measles.md (hunk 2 above)
Comment (kimandrews) [reply static-site/content/blog/2024-06-12-new-resources-for-measles.md]: Done in [5bddcd7](https://github.com/nextstrain/nextstrain.org/pull/911/commits/5bddcd7659cfa2d55712dc2465d39c04bd18291f)

File: static-site/content/blog/2024-06-12-new-resources-for-measles.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+author: "Nextstrain team"
```
Comment (tsibley) [inline static-site/content/blog/2024-06-12-new-resources-for-measles.md:2]:
```suggestion
author: Kim Andrews
```

If you wrote this, it seems you should have the byline! :-)

**Pull request merged**